common/src/definitions/backpacks.ts

```diff
@@ -74,37 +74,36 @@ export const Backpacks = ObjectDefinitions.create<BackpackDefinition>()(
                     medikit: 3,
                     cola: 10,
                     tablets: 3,
-                    "12g": 60,
-                    "556mm": 240,
-                    "762mm": 240,
-                    "9mm": 330,
-                    "127mm": 40,
+                    "12g": 45,
+                    "556mm": 210,
+                    "762mm": 210,
+                    "9mm": 270,
+                    "127mm": 30,
                     power_cell: Infinity,
                     curadell: 3,
-                    frag_grenade: 9,
-                    smoke_grenade: 9
+                    frag_grenade: 8,
+                    smoke_grenade: 8
                 }
             },
             "Regular"
         ),
         apply(
             "backpack_factory",
             {
-                level: 3,
```

Comment:
uh you removed the level from the level 3 backpack

Reply:
fixed, i think

Reply:
uh you set it to 2 not 3 now lmao

Reply:
one sex